### ISSUE TITLE
Missing dependency python in ubuntu deb package

### ISSUE THREAD

**Issue by Belse** [mapcrafter/mapcrafter#270]
I got a configure error when installing mapcrafter from repo "http://packages.mapcrafter.org/ubuntu xenial main"
fail on no python in a script: "#!/usr/bin/env python"... the exact message is no longer available.

Start-Date: 2018-03-12  08:43:41
Commandline: apt install mapcrafter
Requested-By: xxx
Install: libboost-program-options1.58.0:amd64 (1.58.0+dfsg-5ubuntu3.1, automatic), mapcrafter:amd64 (2.4-1), libboost-filesystem1.58.0:amd64 (1.58.0+dfsg-5ubuntu3.1, automatic), libboost-system1.58.0:amd64 (1.58.0+dfsg-5ubuntu3.1, automatic)
Error: Sub-process /usr/bin/dpkg returned an error code (1)
End-Date: 2018-03-12  08:44:01

Of course it goes away after installing python. Minimal OS install here ;)

**Comment by boomshadow** [comment]
Yeah, it was really simple to satisfy the install. You have to have python, and it expects it to be named `python` for the executable. I also needed imagemagick as well to make the install warnings go away:

    sudo apt install python3
    sudo ln -s /usr/bin/python3 /usr/bin/python
    sudo apt install imagemagick

_Then_ I could run the mapcrafter install just fine:

    sudo apt install mapcrafter
